services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -207,7 +207,13 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
         auto_default_from_env=True
     )
 
-    DIRECTOR_V2_DOCKER_REGISTRY: RegistrySettings = Field(auto_default_from_env=True)
+    DIRECTOR_V2_DOCKER_REGISTRY: RegistrySettings = Field(
+        auto_default_from_env=True,
+        description="settings for the private registry deployed with the platform",
+    )
+    DIRECTOR_V2_DOCKER_HUB_REGISTRY: RegistrySettings | None = Field(
+        description="public DockerHub registry settings"
```

Comment:
Q: why there is no "auto_default_from_env" here? what is the difference?

Reply:
so not setting this means that it will only read JSON and not the sub settings field? ok did not know that.